### ISSUE TITLE
Path parameters in "Path Item Object" not correctly validated.

### ISSUE THREAD

**Issue by fabiohecht** [openapi4j/openapi4j#128]
**Describe the bug**
Path parameters in "Path Item Object" not correctly validated.

**To Reproduce**
I tested a schema like the one below, which I believe is a valid one, but the library validation complains about path parameter 'pathParam' in path '/path/{pathParam}' being expected but undefined (code: 124)

```
(...)
/path/{pathParam}
  parameters:
    - name: pathParam
      in: path
      required: true
      description: A path parameter
      schema:
        type: string
  get:
(...)
```

**Expected behavior**
Expected that path parameters can be set in "Path Item Object" and optionally overridden in "Operation Object".

**Comment by llfbandit** [comment]
Did you miss the colon at the end of `/path/{pathParam}` or is it only in your report?
Otherwise, what if you add '/' at the end or a fixed path fragment?

**Comment by fabiohecht** [comment]
Hi, sorry about that, the missing colon is a typo in the report only. I also tried with `/path/{pathParam}/:` and got the same result. Thanks for looking into this.

**Comment by fabiohecht** [comment]
This is the relevant stack trace:
```
	at org.openapi4j.parser.validation.v3.OpenApi3Validator.validate(OpenApi3Validator.java:28)
	at org.openapi4j.parser.OpenApi3Parser.parse(OpenApi3Parser.java:41)
	at org.openapi4j.parser.OpenApi3Parser.parse(OpenApi3Parser.java:17)
	at org.openapi4j.parser.OpenApiParser.parse(OpenApiParser.java:53)
```

using  openapi-operation-validator version 1.0.1

**Comment by llfbandit** [comment]
Thanks for the further details. Can you upgrade to 1.0.3? This should be fixed in 1.0.2.

**Comment by fabiohecht** [comment]
works fine now, thanks!
